src/libraries/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppMinimum);$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>net8.0;$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
I think that breaks source-build as we rely on .NETCoreApp TFMs getting updated to NetCoreAppCurrent when building from source.

I think it would be better to condition `FunctioningBinaryFormatter` differently when building from source.

That said, I just realized that this whole strategy (the OOB functional package) won't work when building from source as then we only build one .NETCoreApp TFM today.

cc @bartonjs @adamsitnik

Reply:
> I think that breaks source-build as we rely on .NETCoreApp TFMs getting updated to NetCoreAppCurrent when building from source.

I'm going to try and see what happens when applying this to the vmr.

> I just realized that this whole strategy (the OOB functional package) won't work when building from source as then we only build one .NETCoreApp TFM today.

Perhaps we don't need the OOB package for source-build.

Reply:
> I'm going to try and see what happens when applying this to the vmr.

You can see that this breaks source-build just by inspecting the failing repo source-build CI leg in this PR. This introduces a new prebuild. Source-build should only build the latest TFM (net9.0) and netstandard2.0/netstandard2.1. With this change, the net8.0 TFM is also built and therefore the Microsoft.NETCore.App.Ref targeting pack is restored.

Reply:
Thanks. I'll change to only building `NetCoreAppCurrent` for source-build.

Reply:
Cool. Just to double check, note that we already only build `NetCoreAppCurrent` when building from source (in addition to .NET Standard) because of this: https://github.com/dotnet/runtime/blob/ddc9d5919711f2eddacaf51979655dcc73f7530a/Directory.Build.props#L104

I think you only need to change one line to fix the issue:

```xml
<FunctioningBinaryFormatter Condition="'$(TargetFramework)' == '$(NetCoreAppMinimum)' and '$(DotNetBuildSourceOnly)' != 'true'">true</FunctioningBinaryFormatter>
```

Reply:
I was just making that change. I had it in two lines. I'll join them together.